src/current/v23.1/changefeed-for.md

```diff
@@ -20,12 +20,12 @@ For more information, see [Change Data Capture Overview](change-data-capture-ove
 ## Required privileges
 
 {{site.data.alerts.callout_info}}
-Starting in v22.2, CockroachDB introduces a new [system-level privilege model](security-reference/authorization.html#supported-privileges) that provides finer control over a user's privilege to work with the database, including creating and managing changefeeds. 
+Starting in v22.2, CockroachDB introduces a new [system-level privilege model](security-reference/authorization.html#supported-privileges) that provides finer control over a user's privilege to work with the database, including creating and managing changefeeds.
```

Comment:
```suggestion
In v22.2 and above, CockroachDB introduces a new [system-level privilege model](security-reference/authorization.html#supported-privileges) that provides more fine-grained control over a user's privilege to work with the database, including the ability to create and manage changefeeds.
```
By "the database" do you mean the cluster?

The "including creating..." seems like a bit of a dangling modifier, since the main verb in the sentence is "introduces" and the sub-clause verb is "provides". I've rewritten it a bit. WDYT?

Reply:
Yeah that's fine, this isn't something that I wrote personally, so lgtm